pom.xml

```diff
@@ -76,8 +76,11 @@
         <!-- Don't change this version without also changing the shiro-aspect and shiro-features
              modules' OSGi metadata: -->
         <aspectj.version>1.8.9</aspectj.version>
+        <cas.client.core.version>3.2.2</cas.client.core.version>
         <commons.cli.version>1.3.1</commons.cli.version>
         <commons.codec.version>1.10</commons.codec.version>
+        <commons.configuration2.version>2.1</commons.configuration2.version>
```

Comment:
Version 2.2 of commons-configuration2 has the springbeans package imports as optional.